.bazelrc

```diff
@@ -1,28 +1,72 @@
-# Android configs.
+# Make Bazel print out all options from rc files.
+build --announce_rc
+
+build --config=short_logs
+
+# This flag is required by tensorflow
+common --experimental_repo_remote_exec
+
+# Default options should come above this line.
+
+# Configure logs
+build:short_logs --output_filter=DONT_MATCH_ANYTHING
+build:verbose_logs --output_filter=
+
+# Suppress C++ compiler warnings, otherwise build logs become 10s of MBs.
+build:android --copt=-w
+build:ios --copt=-w
+build:windows --copt=/W0
+
+# Build in C++ 14 mode.
+build:android --cxxopt=-std=c++14
+build:android --host_cxxopt=-std=c++14
+build:ios --cxxopt=-std=c++14
+build:ios --host_cxxopt=-std=c++14
+build:windows --cxxopt=/std:c++14
+build:windows --host_cxxopt=/std:c++14
+
+# Android configs
 build:android --crosstool_top=//external:android/crosstool
 build:android --host_crosstool_top=@bazel_tools//tools/cpp:toolchain
-
 build:android_arm64 --config=android
 build:android_arm64 --cpu=arm64-v8a
 build:android_arm64 --fat_apk_cpu=arm64-v8a
```

Comment:
Do we need a fat APK anymore if it only contains one arch?

Reply:
According to Tensorflow config we do need both flags:
https://github.com/tensorflow/tensorflow/blob/master/.bazelrc#L158-L160
```bazel
# Android configs. Bazel needs to have --cpu and --fat_apk_cpu both set to the
# target CPU to build transient dependencies correctly. See
# https://docs.bazel.build/versions/master/user-manual.html#flag--fat_apk_cpu
```